doc/JSON_FLAGS.md

```diff
@@ -944,8 +944,11 @@ Those flags are added by the game code to specific items (that specific welder,
 - ```FISHING_BASIC``` Use a fishing rod
 - ```JET_INJECTOR``` Inject some jet drugs right into your veins.
 - ```CABLE_ATTACH``` This item is a cable spool. Use it to try to attach to a vehicle.
-
-
+- ```CAPTURE_MONSTER_ACT``` Capture and encapsulate a monster. This action is affected by several flags which as of this point, have no other purposes. The associated action is also used for releasing it.
+  - ```NO_BREAK_ON_FAIL``` Prevents a monster capturing device, like a pet carrier, from breaking if it fails to capture the monster.
```

Comment:
It is used in the capture_monster_act function. Would you propose that I change the flag used to ```UNBREAKABLE_MELEE``` 
That seems close, but not quite right. 
The capture_monster_act function breaks the item if it decides that it fails and does not have the flag.